loaders/playgroundLoader.js

```diff
@@ -0,0 +1,125 @@
+module.exports = function (lang) {
+  return function playgroundLoader(source) {
+    const { imports, scopes } = parseScope(source)
+    const code = JSON.stringify(parseCode(source))
+    const desc = parseStringAfterToken(source, lang === 'en' ? 'descEN' : 'descZH')
+    const title = parseStringAfterToken(source, lang === 'en' ? 'titleEN' : 'titleZH')
+    const s = `
+        import React from 'react'
+        import { Playground } from 'lib/components'
+        ${imports}
+        export default function App(){
+          return <Playground
+          scope={{${scopes}}}
+          title={${title}}
+          desc={${desc}}
+          code={${code}}
+        />
+        }`
+    return s
+  }
+}
+
+function parseScope(str) {
+  const imports = str.split('\n').filter(line => line.startsWith('import'))
+  const scopes = imports
+    .map(line => {
+      const start = line.indexOf('{')
+      const end = line.indexOf('}')
+      return line.slice(start + 2, end).split(',')
+    })
+    .flat()
+    .join(',')
+
+  return { imports: imports.join('\n'), scopes }
+}
```

Comment:
support `alias`?

Reply:
react live `scope` prop already injected `React`

Reply:
you mean `import {A as AA} from 'aa'`? Not supported
I will try to exclude React from scope to see what happen

Reply:
Just notice that the `import` inside `{}` can be parsed out properly
Even `import A from 'a'` is not supported
However, we use `playground loader` as a code demo only
`import {AA} from 'component'` is enough